### DISCUSSION TITLE
移动端顶部元素过多导致样式异常

### DISCUSSION THREAD

**Discussion by opoet** [spencerwooo/onedrive-vercel-index#427] [General]
当顶部元素过多时会导致样式异常

![image](https://user-images.githubusercontent.com/30021379/153752037-dd16eca3-69e9-4d6d-b9b4-a288b72d2476.png)

**Comment by myl7** [comment]
或许 links 在空间不足时可以做成 dropdown？ cc @spencerwooo

**Reply by spencerwooo** [reply]
小问题，后面修（其实把 Search 和右边 Logout 在 `md:` 的时候变成一个 icon 就应该够了）

**Reply by opoet** [reply]
但是不排除有用户添加很多的 links 的情况，似乎 dropdown 更加稳妥 ？

**Comment by spencerwooo** [comment]
这个也修了，在 mobile 上会隐藏部分文字，dropdown 后面再议。
